### DISCUSSION TITLE
Copilot makes Intellij freeze when creating a test class

### DISCUSSION THREAD

**Discussion by martinhansson80** [community/community#38284] [Copilot Conversations]
Hi.

When running the Beta version of Intellij for Linux (223.7401.7) Copilot makes Intellij freeze when I try to create a new Java test class.
I'm trying to create the test class using the shortcut "Ctrl + Shift + T" and after I've confirmed the name/location, the IDE freezes. 

It works great in Intellij 2022.2.3 (222.4345.14).

Anyone had the same issue?

**Comment by ossanna16** [comment]
Hi there @mha109 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
